frontend/components/Teams/TeamApp.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Link } from "react-router-dom";
+
+export default class TeamApp extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { teams: [] };
+  }
+
+  componentDidMount() {
+    fetch('/api/v1/teams').
+      then((response) => response.json()).
+      then((teams) =>  this.setState({ teams }));
+  }
+
+  render() {
+    const { teams } = this.state;
+    return (
+      <div>
+        <h3>All teams</h3>
+        <table>
+          <thead>
+            <tr>
+              <th>Team name</th>
+            </tr>
+          </thead>
+          <tbody>
+            {
+              teams.map((team) => {
+                return (
+                  <tr key={team.id}>
+                    <td>
+                      <Link to={`/teams/${team.id}`}>
```

Comment:
We agreed that it's the one page, where should be team names and team members. So we don't need to use `Link` here. Here should be a component with team leads and members. 
Add hook/state for `activeTeam`. initially, it's empty after a user clicks on some team name, we should show this team info on this page 
ex.
{activeTeam && <SepareteTeam props={props}>}